docs/releases/1.7.txt

```diff
@@ -637,7 +637,7 @@ Forms
   value.
 
 * :attr:`SelectDateWidget.months
-  <django.forms.extras.widgets.SelectDateWidget.months>` can be used to
+  <django.forms.SelectDateWidget.months>` can be used to
```

Comment:
You're right, I missed the attr role. in that case I'd go with plain  \`\`django.forms.extras.widgets.SelectDateWidget.months\`\` or "the \`\`months\`\` attribute of \`\`django.forms.extras.widgets.SelectDateWidget.months\`\` [...]".